Holocene Standard Rollup Charter (#37)
ethereum-optimism/OPerating-manual: Standard Rollup Charter.md

### Version Validation
The most important onchain criteria is that a chain be on a standard, governance-approved release of the OP Stack. This check is performed by comparing all bytecode for the chain’s L1 smart contracts to the standard bytecode corresponding to a governance-approved release of the OP Stack.

> Version validation is a strict, critical requirement. To securely hand over upgradability to the Collective, a chain’s L1 smart contracts must match the release tags defined by [this TOML file](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-versions-mainnet.toml) in the Superchain Registry. This corresponds to `op-contracts@v1.8.0-rc.3`.

Author: ```suggestion
Version validation is a strict, critical requirement. To securely hand over upgradability to the Collective, a chain’s L1 smart contracts must match the release tags defined by [this TOML file](https://github.com/ethereum-optimism/superchain-registry/blob/9095778d45a5066649890ee838f87b27062a0d4d/validation/standard/standard-versions-mainnet.toml) in the Superchain Registry. This corresponds to `op-contracts@v1.8.0-rc.3`.
```

> Version validation is a strict, critical requirement. To securely hand over upgradability to the Collective, a chain’s L1 smart contracts must match the release tags defined by [this TOML file](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-versions-mainnet.toml) in the Superchain Registry. At the time of writing, this corresponds to `op-contracts@v1.6.0`.

Author: ```suggestion
Version validation is a strict, critical requirement. To securely hand over upgradability to the Collective, a chain’s L1 smart contracts must match the release tags defined by [this TOML file](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-versions-mainnet.toml) in the Superchain Registry. At the time of writing, this corresponds to `op-contracts@v1.8.0-rc.3`.
```

For those interested, the code currently used in the Superchain Registry to perform version (and configuration) checks can be found[ here](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/validation_test.go). The Optimism Foundation may, from time to time, update this code (e.g. for quality-of-life improvements or other refactors), **so long as it does not violate the semantic interpretation of the above TOML files, which are subject to Governance approval**.

### Configuration Check
Beyond being on a standard version of the OP Stack, all configuration values for the chain must be within high-security, well-tested bounds, and all administrative roles must be  set correctly. There are two main components:
* **Parameter Configuration**: A set of requirements for the “protocol parameters” of a chain — things like block time, gas metering, and other low-level variables — to either equal a certain value, or fall within a specified range. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-config-params-mainnet.toml).

> * **Role Configuration**: A set of requirements for the privileged administrative roles for the chain. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-config-roles-mainnet.toml). Primarily, these checks involve ensuring that the Security Council holds authorization to perform upgrades, and other minor Stage 1 requirements (see [here](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157)).

Reviewer: This looks good to me.

> * **Parameter Configuration**: A set of requirements for the “protocol parameters” of a chain — things like block time, gas metering, and other low-level variables — to either equal a certain value, or fall within a specified range. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-config-params-mainnet.toml).
> * **Role Configuration**: A set of requirements for the privileged administrative roles for the chain. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/a3a7f2e1df081125e3d1fe9ff4a24ff5aaa238a8/validation/standard/standard-config-roles-mainnet.toml). Primarily, these checks involve ensuring that the Security Council holds authorization to perform upgrades, and other minor Stage 1 requirements (see [here](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157)).

Author: ```suggestion
* **Parameter Configuration**: A set of requirements for the “protocol parameters” of a chain — things like block time, gas metering, and other low-level variables — to either equal a certain value, or fall within a specified range. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/9095778d45a5066649890ee838f87b27062a0d4d/validation/standard/standard-config-params-mainnet.toml).
* **Role Configuration**: A set of requirements for the privileged administrative roles for the chain. The specific requirements can be found [here](https://github.com/ethereum-optimism/superchain-registry/blob/9095778d45a5066649890ee838f87b27062a0d4d/validation/standard/standard-config-roles-mainnet.toml). Primarily, these checks involve ensuring that the Security Council holds authorization to perform upgrades, and other minor Stage 1 requirements (see [here](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157)).
```
